Samplero/Samplero/Global/Extension/UIViewController+Alert.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIViewController+Alert.swift
+//  Samplero
+//
+//  Created by Minkyeong Ko on 2022/10/12.
+//
+
+import UIKit
+
+extension UIViewController {
+    func makeAlert(title: String,
+                   message: String,
+                   okAction: ((UIAlertAction) -> Void)? = nil,
+                   completion : (() -> Void)? = nil) {
+        let alertViewController = UIAlertController(title: title,
+                                                    message: message,
+                                                    preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "확인", style: .default, handler: okAction)
```

Comment:
p5.

```suggestion
        let okayAction = UIAlertAction(title: "확인", style: .default, handler: okayAction)
```

okayAction이 축약어를 지양하여 쪼금 더 나은 거 같습니다

Reply:
그렇네요!! 변경했습니다:)